Makefile

```diff
@@ -5,7 +5,7 @@ MAKEFLAGS += --no-builtin-variables
 COVERAGE_FILE ?= .coverage
 
 # allow overriding the name of the venv directory
-VENV_DIR ?= venv
+VENV_DIR ?= .venv
```

Comment:
The `venv` make target on line 32 should also be renamed to `.venv`, as that is now what `uv sync` will create

Reply:
I tested it locally and `make venv` still creates a `.venv`

Reply:
so do we want this suggestion?